site/tests/integration/login-flow.test.ts

```diff
@@ -0,0 +1,201 @@
+import execa from "execa";
+import { expect, test } from "playwright-test-coverage";
+
+import { readValueFromRecentDummyEmail } from "../shared/dummy-emails";
+import { openLoginModal } from "../shared/nav";
+
+const emailInputSelector = '[placeholder="claude\\@example\\.com"]';
+const loginButtonSelector = "button[type=submit]:has-text('Log In')";
+const verificationCodeInputSelector = '[placeholder="your-verification-code"]';
+const accountDropdownButtonSelector = '[data-testid="account-dropdown-button"]';
+
+test("login works for an existing user (via verification code)", async ({
+  page,
+  isMobile,
+}) => {
+  await execa("yarn", ["exe", "scripts/seed-db.ts"]);
```

Comment:
Let's use [test.beforeEach](https://playwright.dev/docs/api/class-test#test-before-each) for this

Reply:
I can do that in a follow-up 👍

Reply:
Done in https://github.com/blockprotocol/blockprotocol/pull/455